serialization/src/ATMMetaForceProxy.cpp

```diff
@@ -0,0 +1,80 @@
+#include "ATMMetaForceProxy.h"
+#include "ATMMetaForce.h"
+
+#include <vector>
+
+using namespace ATMMetaForcePlugin;
+
+ATMMetaForceProxy::ATMMetaForceProxy() : OpenMM::SerializationProxy("ATMMetaForce") {
+}
+
+void ATMMetaForceProxy::serialize(const void* object, OpenMM::SerializationNode& node) const {
+    node.setIntProperty("version", 0);
+    const ATMMetaForce& force = *reinterpret_cast<const ATMMetaForce*>(object);
+
+    node.setIntProperty("forceGroup", force.getForceGroup());
+    node.setStringProperty("name", force.getName());
+
+    node.setDoubleProperty("lambda1", force.getDefaultLambda1());
+    node.setDoubleProperty("lambda2", force.getDefaultLambda2());
+    node.setDoubleProperty("alpha", force.getDefaultAlpha());
+    node.setDoubleProperty("u0", force.getDefaultU0());
+    node.setDoubleProperty("w0", force.getDefaultW0());
+    node.setDoubleProperty("uMax", force.getDefaultUmax());
+    node.setDoubleProperty("ubCore", force.getDefaultUbcore());
+    node.setDoubleProperty("aCore", force.getDefaultAcore());
+    node.setDoubleProperty("direction", force.getDefaultDirection());
+
+    OpenMM::SerializationNode& variableForceGroups = node.createChildNode("GlobalParameters");
+    for (const auto i : force.getVariableForceGroups()) {
+        variableForceGroups.createChildNode("Parameter").setIntProperty("group", i);
+    }
```

Comment:
You might want to give this a more accurate name, such as "VariableForceGroups".